features/us20_borrower_views_deed.feature

```diff
@@ -17,7 +17,9 @@ Scenario: Borrower enters a valid deed reference
          - e-MD ref
 
   When I navigate to the borrower frontend "/searchdeed" page
-  And I search for the created deed
+  When I retrieve the deed id
```

Comment:
Minor thing but should probably change this to **And** I retrieve the deed id as I don't think we shouldn't have move than one of each Given When Then in a scenario, the rest should be And.